Update /5/ urls (#741)
plone/training: docs/mastering-plone/code.md

```

The names of the tags are the same as the URL of the chapter.

> The tag for the chapter {doc}`mastering-plone/registry` is `registry`.

Reviewer: ```suggestion
The tag for the chapter {doc}`/mastering-plone/registry` is `registry`.
```

> The tag for the chapter <https://training.plone.org/mastering-plone/registry.html> is `registry`

Reviewer: ```suggestion
The tag for the chapter {doc}`mastering-plone/registry` is `registry`.
```

> and you can get it with {command}`git checkout registry`.

Reviewer: ```suggestion
You can get it with {command}`git checkout registry`.
```
